command/base_flags.go

```diff
@@ -246,9 +247,30 @@ func (i *intValue) Set(s string) error {
 	if err != nil {
 		return err
 	}
+	switch strconv.IntSize {
```

Comment:
I would rather see you define a local constant somewhere, e.g. in package `consts`.  We might have to use it again elsewhere someday.  Recall I suggested in slack `const MaxInt = int(^uint(0) >> 1)`.

Reply:
Yes, I used it in barrier_aes_gcm.go but in this case I got an error when I compared int64 with int. I was not sure about converting the input value to int to compare with MaxInt, since it could get truncated in case of 32 bit and we wont be able to check overflow. I did not get that error when I used math package

Reply:
You can cast `int64(MaxInt)` safely, because an int64 is guaranteed not to be smaller than the local int word size, and MaxInt is defined such that it will always fit in the local int word size.